src/data/light.json

```diff
@@ -3,19 +3,23 @@
     "text": {
       "emphasis": {
         "value": "{colors.grey.700}",
-        "type": "color"
+        "type": "color",
+        "description": "Used for any text that has font weight of 700 and up"
```

Comment:
For this, I'm not sure if we want to tie this to useage of weights? Something to discuss at team meeting

Reply:
Updated to remove font weight references!